### PR TITLE
Enhancement: Flatten directory structure

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] flattens the directory structure following the switch to PSR-4

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,12 +26,12 @@
     },
     "autoload": {
         "psr-4": {
-          "Qandidate\\Toggle\\": "lib/Qandidate/Toggle/"
+          "Qandidate\\Toggle\\": "lib/"
```
Comment (localheinz) [inline composer.json]: @wjzijderveld 

What do you think about changing the directory name from `lib` to `src`?

**Comment by asm89** [comment]
:-1: Git history will be b0rked and the gains are minimal imo.

**Comment by wjzijderveld** [comment]
I don't like this either, it doesn't provide added benefits.
And like @asm89 said, it does screw up the history badly :)

**Comment by localheinz** [comment]
@asm89 @wjzijderveld 

No worries!

**Pull request closed without merging**